client/server.go

```diff
@@ -334,6 +341,23 @@ func (server *CqlServer) BindAndInit(
 	}
 }
 
+type Response struct {
+	responseFrame *frame.Frame
+	rawResponse   []byte
+}
+
+func NewFrameResponse(frameResponse *frame.Frame) *Response {
```

Comment:
Same for the constructor functions, it seems they are not meant to be called from outside this package.

Reply:
👍  done